packages/clerk-js/src/core/fapiClient.ts

```diff
@@ -197,8 +197,15 @@ export default function createFapiClient(clerkInstance: Omit<Clerk, 'proxyUrl'>)
     } catch (e) {
       clerkNetworkError(urlStr, e);
     }
+
+    const responseHeaders = {
+      country: response.headers.get('x-country'),
+    };
+
     const json: FapiResponseJSON<T> = await response.json();
-    const fapiResponse: FapiResponse<T> = Object.assign(response, { payload: json });
+    const fapiResponse: FapiResponse<T> = Object.assign(response, {
+      payload: { ...json, response_headers: responseHeaders },
```

Comment:
It looks like we're adding the `response_headers` for all requests.
If so, we need to also update the `FapiResponse` and `FapiResponse` to reflect this.

Reply:
On a second thought, I see that `FapiResponseJSON` includes an optional `meta` key: https://github.com/clerkinc/javascript/blob/ec2262459d34f1a443d272dfa405d117d97251ae/packages/clerk-js/src/core/fapiClient.ts#L43

Can we maybe reuse that instead of adding the `response_headers` object? Eg `payload.meta.country`